Can we build a Perl to support threads in termux?

```
       ╲ ▁▂▂▂▁ ╱        
       ▄███████▄        
      ▄██ ███ ██▄        HUAWEI_P30_Pro-78fc437999
     ▄███████████▄       OS: Android 10
  ▄█ ▄▄▄▄▄▄▄▄▄▄▄▄▄ █▄    Device: VOG-AL10 (HWVOG)
  ██ █████████████ ██    ROM: VOG-AL10 10.1.0.162(C00E160R1P8)
  ██ █████████████ ██    Baseband: unknown
  ██ █████████████ ██    Kernel: aarch64 Linux 4.14.116
  ██ █████████████ ██    Uptime: 
     █████████████       CPU: Hisilicon Kirin980
      ███████████        GPU: Hisilicon Kirin980
       ██     ██         RAM: 4523MiB / 7593MiB
       ██     ██        
```


I tried to run an app but it raised the error:

```
This Perl not built to support threads
```

So, I download the `perl-5.16.1` and build it with 
```bash
./Configure -des -Dprefix=data/data/com.termux/files/usr/local/perl -Dusethreads -Uinstalluserbinperl -Dcc=gcc
```
`Configure` is fine:
```text
Finding dependencies for walk.o.
make[2]: Entering directory '/data/data/com.termux/files/home/Softwarw/perl-5.16.1/x2p'
echo Makefile.SH cflags.SH | tr ' ' '\n' >.shlist
make[2]: Leaving directory '/data/data/com.termux/files/home/Softwarw/perl-5.16.1/x2p'
Updating makefile...
make[1]: Leaving directory '/data/data/com.termux/files/home/Softwarw/perl-5.16.1/x2p'
Now you must run 'make'.

If you compile perl5 on a different machine or from a different object
directory, copy the Policy.sh file from this object directory to the
new one before you run Configure -- this will help you with most of
the policy defaults.
```

Bug wen I run make:

```
make[1]: Leaving directory '/data/data/com.termux/files/home/Softwarw/perl-5.16.1/cpan/CPAN-Meta'
./miniperl -Ilib make_ext.pl cpan/CPAN-Meta-YAML/pm_to_blib MAKE=make LIBPERL_A=libperl.a
	Making CPAN::Meta::YAML (all)
make[1]: Entering directory '/data/data/com.termux/files/home/Softwarw/perl-5.16.1/cpan/CPAN-Meta-YAML'
Makefile out-of-date with respect to ../../lib/Config.pm ../../config.h
Cleaning current config before rebuilding Makefile...
make -f Makefile.old clean > /dev/null 2>&1
../../miniperl "-I../../lib" "-I../../lib" Makefile.PL "INSTALLDIRS=perl" "INSTALLMAN1DIR=none" "INSTALLMAN3DIR=none" "PERL_CORE=1" "LIBPERL_A=libperl.a"
Writing Makefile for CPAN::Meta::YAML
==> Your Makefile has been rebuilt. <==
==> Please rerun the make command.  <==
false
make[1]: *** [Makefile:728: Makefile] Error 1
make[1]: Leaving directory '/data/data/com.termux/files/home/Softwarw/perl-5.16.1/cpan/CPAN-Meta-YAML'
 make config PERL_CORE=1 LIBPERL_A=libperl.a failed, continuing anyway...
Making all in cpan/CPAN-Meta-YAML
 make all PERL_CORE=1 LIBPERL_A=libperl.a
make[1]: Entering directory '/data/data/com.termux/files/home/Softwarw/perl-5.16.1/cpan/CPAN-Meta-YAML'
Skip ../../lib/CPAN/Meta/YAML.pm (unchanged)
make[1]: Leaving directory '/data/data/com.termux/files/home/Softwarw/perl-5.16.1/cpan/CPAN-Meta-YAML'
./miniperl -Ilib make_ext.pl cpan/CPANPLUS/pm_to_blib MAKE=make LIBPERL_A=libperl.a
	Making CPANPLUS (all)
make[1]: Entering directory '/data/data/com.termux/files/home/Softwarw/perl-5.16.1/cpan/CPANPLUS'
Makefile out-of-date with respect to ../../lib/Config.pm ../../config.h
Cleaning current config before rebuilding Makefile...
make -f Makefile.old clean > /dev/null 2>&1
../../miniperl "-I../../lib" "-I../../lib" Makefile.PL "INSTALLDIRS=perl" "INSTALLMAN1DIR=none" "INSTALLMAN3DIR=none" "PERL_CORE=1" "LIBPERL_A=libperl.a"
Writing Makefile for CPANPLUS
==> Your Makefile has been rebuilt. <==
==> Please rerun the make command.  <==
false
make[1]: *** [Makefile:899: Makefile] Error 1
make[1]: Leaving directory '/data/data/com.termux/files/home/Softwarw/perl-5.16.1/cpan/CPANPLUS'
 make config PERL_CORE=1 LIBPERL_A=libperl.a failed, continuing anyway...
Making all in cpan/CPANPLUS
 make all PERL_CORE=1 LIBPERL_A=libperl.a
make[1]: Entering directory '/data/data/com.termux/files/home/Softwarw/perl-5.16.1/cpan/CPANPLUS'
Skip ../../lib/CPANPLUS/Backend/RV.pm (unchanged)
Skip ../../lib/CPANPLUS/Shell/Default.pm (unchanged)
Skip ../../lib/CPANPLUS/Module.pm (unchanged)
Skip ../../lib/CPANPLUS/Module/Author.pm (unchanged)
Skip ../../lib/CPANPLUS/Module/Checksums.pm (unchanged)
Skip ../../lib/CPANPLUS/Dist/MM.pm (unchanged)
Skip ../../lib/CPANPLUS/Internals.pm (unchanged)
Skip ../../lib/CPANPLUS/Module/Signature.pm (unchanged)
Skip ../../lib/CPANPLUS/Hacking.pod (unchanged)
Skip ../../lib/CPANPLUS.pm (unchanged)
Skip ../../lib/CPANPLUS/Shell/Default/Plugins/Remote.pm (unchanged)
Skip ../../lib/CPANPLUS/Backend.pm (unchanged)
Skip ../../lib/CPANPLUS/Internals/Report.pm (unchanged)
Skip ../../lib/CPANPLUS/Module/Author/Fake.pm (unchanged)
Skip ../../lib/CPANPLUS/FAQ.pod (unchanged)
Skip ../../lib/CPANPLUS/Config.pm (unchanged)
Skip ../../lib/CPANPLUS/Internals/Utils.pm (unchanged)
Skip ../../lib/CPANPLUS/Shell/Default/Plugins/Source.pm (unchanged)
Skip ../../lib/CPANPLUS/Internals/Constants/Report.pm (unchanged)
Skip ../../lib/CPANPLUS/Shell.pm (unchanged)
Skip ../../lib/CPANPLUS/Configure/Setup.pm (unchanged)
Skip ../../lib/CPANPLUS/Internals/Source.pm (unchanged)
Skip ../../lib/CPANPLUS/Internals/Fetch.pm (unchanged)
Skip ../../lib/CPANPLUS/Shell/Classic.pm (unchanged)
Skip ../../lib/CPANPLUS/Dist.pm (unchanged)
Skip ../../lib/CPANPLUS/Configure.pm (unchanged)
Skip ../../lib/CPANPLUS/Selfupdate.pm (unchanged)
Skip ../../lib/CPANPLUS/Internals/Search.pm (unchanged)
Skip ../../lib/CPANPLUS/Internals/Extract.pm (unchanged)
Skip ../../lib/CPANPLUS/Internals/Source/SQLite/Tie.pm (unchanged)
Skip ../../lib/CPANPLUS/Internals/Utils/Autoflush.pm (unchanged)
Skip ../../lib/CPANPLUS/Shell/Default/Plugins/CustomSource.pm (unchanged)
Skip ../../lib/CPANPLUS/Dist/Sample.pm (unchanged)
Skip ../../lib/CPANPLUS/Error.pm (unchanged)
Skip ../../lib/CPANPLUS/Module/Fake.pm (unchanged)
Skip ../../lib/CPANPLUS/Internals/Source/SQLite.pm (unchanged)
Skip ../../lib/CPANPLUS/Internals/Source/Memory.pm (unchanged)
Skip ../../lib/CPANPLUS/Dist/Autobundle.pm (unchanged)
Skip ../../lib/CPANPLUS/Internals/Constants.pm (unchanged)
Skip ../../lib/CPANPLUS/Dist/Base.pm (unchanged)
Skip ../../lib/CPANPLUS/Shell/Default/Plugins/HOWTO.pod (unchanged)
cp bin/cpanp-run-perl blib/script/cpanp-run-perl
/data/data/com.termux/files/home/Softwarw/perl-5.16.1/cpan/CPANPLUS/../../miniperl "-I../../lib" "-I../../lib" -MExtUtils::MY -e 'MY->fixin(shift)' -- blib/script/cpanp-run-perl
cp bin/cpan2dist blib/script/cpan2dist
/data/data/com.termux/files/home/Softwarw/perl-5.16.1/cpan/CPANPLUS/../../miniperl "-I../../lib" "-I../../lib" -MExtUtils::MY -e 'MY->fixin(shift)' -- blib/script/cpan2dist
cp bin/cpanp blib/script/cpanp
/data/data/com.termux/files/home/Softwarw/perl-5.16.1/cpan/CPANPLUS/../../miniperl "-I../../lib" "-I../../lib" -MExtUtils::MY -e 'MY->fixin(shift)' -- blib/script/cpanp
make[1]: Leaving directory '/data/data/com.termux/files/home/Softwarw/perl-5.16.1/cpan/CPANPLUS'
./miniperl -Ilib make_ext.pl cpan/CPANPLUS-Dist-Build/pm_to_blib MAKE=make LIBPERL_A=libperl.a
	Making CPANPLUS::Dist::Build (all)
make[1]: Entering directory '/data/data/com.termux/files/home/Softwarw/perl-5.16.1/cpan/CPANPLUS-Dist-Build'
Makefile out-of-date with respect to ../../lib/Config.pm ../../config.h
Cleaning current config before rebuilding Makefile...
make -f Makefile.old clean > /dev/null 2>&1
../../miniperl "-I../../lib" "-I../../lib" Makefile.PL "INSTALLDIRS=perl" "INSTALLMAN1DIR=none" "INSTALLMAN3DIR=none" "PERL_CORE=1" "LIBPERL_A=libperl.a"
Writing Makefile for CPANPLUS::Dist::Build
==> Your Makefile has been rebuilt. <==
==> Please rerun the make command.  <==
false
make[1]: *** [Makefile:731: Makefile] Error 1
make[1]: Leaving directory '/data/data/com.termux/files/home/Softwarw/perl-5.16.1/cpan/CPANPLUS-Dist-Build'
 make config PERL_CORE=1 LIBPERL_A=libperl.a failed, continuing anyway...
Making all in cpan/CPANPLUS-Dist-Build
 make all PERL_CORE=1 LIBPERL_A=libperl.a
make[1]: Entering directory '/data/data/com.termux/files/home/Softwarw/perl-5.16.1/cpan/CPANPLUS-Dist-Build'
Skip ../../lib/CPANPLUS/Dist/Build.pm (unchanged)
Skip ../../lib/CPANPLUS/Dist/Build/Constants.pm (unchanged)
make[1]: Leaving directory '/data/data/com.termux/files/home/Softwarw/perl-5.16.1/cpan/CPANPLUS-Dist-Build'
./miniperl -Ilib make_ext.pl dist/Carp/pm_to_blib MAKE=make LIBPERL_A=libperl.a
	Making Carp (all)
make[1]: Entering directory '/data/data/com.termux/files/home/Softwarw/perl-5.16.1/dist/Carp'
Makefile out-of-date with respect to ../../lib/Config.pm ../../config.h
Cleaning current config before rebuilding Makefile...
make -f Makefile.old clean > /dev/null 2>&1
../../miniperl "-I../../lib" "-I../../lib" Makefile.PL "INSTALLDIRS=perl" "INSTALLMAN1DIR=none" "INSTALLMAN3DIR=none" "PERL_CORE=1" "LIBPERL_A=libperl.a"
Writing Makefile for Carp
==> Your Makefile has been rebuilt. <==
==> Please rerun the make command.  <==
false
make[1]: *** [Makefile:732: Makefile] Error 1
make[1]: Leaving directory '/data/data/com.termux/files/home/Softwarw/perl-5.16.1/dist/Carp'
 make config PERL_CORE=1 LIBPERL_A=libperl.a failed, continuing anyway...
Making all in dist/Carp
 make all PERL_CORE=1 LIBPERL_A=libperl.a
make[1]: Entering directory '/data/data/com.termux/files/home/Softwarw/perl-5.16.1/dist/Carp'
Skip ../../lib/Carp.pm (unchanged)
Skip ../../lib/Carp/Heavy.pm (unchanged)
make[1]: Leaving directory '/data/data/com.termux/files/home/Softwarw/perl-5.16.1/dist/Carp'
./miniperl -Ilib make_ext.pl dist/Devel-SelfStubber/pm_to_blib MAKE=make LIBPERL_A=libperl.a
	Making Devel::SelfStubber (all)
make[1]: Entering directory '/data/data/com.termux/files/home/Softwarw/perl-5.16.1/dist/Devel-SelfStubber'
Makefile out-of-date with respect to ../../lib/Config.pm ../../config.h
Cleaning current config before rebuilding Makefile...
make -f Makefile.old clean > /dev/null 2>&1
../../miniperl "-I../../lib" "-I../../lib" Makefile.PL "INSTALLDIRS=perl" "INSTALLMAN1DIR=none" "INSTALLMAN3DIR=none" "PERL_CORE=1" "LIBPERL_A=libperl.a"
Writing Makefile for Devel::SelfStubber
==> Your Makefile has been rebuilt. <==
==> Please rerun the make command.  <==
false
make[1]: *** [Makefile:728: Makefile] Error 1
make[1]: Leaving directory '/data/data/com.termux/files/home/Softwarw/perl-5.16.1/dist/Devel-SelfStubber'
 make config PERL_CORE=1 LIBPERL_A=libperl.a failed, continuing anyway...
Making all in dist/Devel-SelfStubber
 make all PERL_CORE=1 LIBPERL_A=libperl.a
make[1]: Entering directory '/data/data/com.termux/files/home/Softwarw/perl-5.16.1/dist/Devel-SelfStubber'
Skip ../../lib/Devel/SelfStubber.pm (unchanged)
make[1]: Leaving directory '/data/data/com.termux/files/home/Softwarw/perl-5.16.1/dist/Devel-SelfStubber'
./miniperl -Ilib make_ext.pl cpan/Digest/pm_to_blib MAKE=make LIBPERL_A=libperl.a
	Making Digest (all)
make[1]: Entering directory '/data/data/com.termux/files/home/Softwarw/perl-5.16.1/cpan/Digest'
Makefile out-of-date with respect to ../../lib/Config.pm ../../config.h
Cleaning current config before rebuilding Makefile...
make -f Makefile.old clean > /dev/null 2>&1
../../miniperl "-I../../lib" "-I../../lib" Makefile.PL "INSTALLDIRS=perl" "INSTALLMAN1DIR=none" "INSTALLMAN3DIR=none" "PERL_CORE=1" "LIBPERL_A=libperl.a"
Writing Makefile for Digest
==> Your Makefile has been rebuilt. <==
==> Please rerun the make command.  <==
false
make[1]: *** [Makefile:735: Makefile] Error 1
make[1]: Leaving directory '/data/data/com.termux/files/home/Softwarw/perl-5.16.1/cpan/Digest'
 make config PERL_CORE=1 LIBPERL_A=libperl.a failed, continuing anyway...
Making all in cpan/Digest
 make all PERL_CORE=1 LIBPERL_A=libperl.a
make[1]: Entering directory '/data/data/com.termux/files/home/Softwarw/perl-5.16.1/cpan/Digest'
Skip ../../lib/Digest.pm (unchanged)
Skip ../../lib/Digest/file.pm (unchanged)
Skip ../../lib/Digest/base.pm (unchanged)
make[1]: Leaving directory '/data/data/com.termux/files/home/Softwarw/perl-5.16.1/cpan/Digest'
./miniperl -Ilib make_ext.pl dist/Dumpvalue/pm_to_blib MAKE=make LIBPERL_A=libperl.a
	Making Dumpvalue (all)
make[1]: Entering directory '/data/data/com.termux/files/home/Softwarw/perl-5.16.1/dist/Dumpvalue'
Makefile out-of-date with respect to ../../lib/Config.pm ../../config.h
Cleaning current config before rebuilding Makefile...
make -f Makefile.old clean > /dev/null 2>&1
../../miniperl "-I../../lib" "-I../../lib" Makefile.PL "INSTALLDIRS=perl" "INSTALLMAN1DIR=none" "INSTALLMAN3DIR=none" "PERL_CORE=1" "LIBPERL_A=libperl.a"
Writing Makefile for Dumpvalue
==> Your Makefile has been rebuilt. <==
==> Please rerun the make command.  <==
false
make[1]: *** [Makefile:728: Makefile] Error 1
make[1]: Leaving directory '/data/data/com.termux/files/home/Softwarw/perl-5.16.1/dist/Dumpvalue'
 make config PERL_CORE=1 LIBPERL_A=libperl.a failed, continuing anyway...
Making all in dist/Dumpvalue
 make all PERL_CORE=1 LIBPERL_A=libperl.a
make[1]: Entering directory '/data/data/com.termux/files/home/Softwarw/perl-5.16.1/dist/Dumpvalue'
Skip ../../lib/Dumpvalue.pm (unchanged)
make[1]: Leaving directory '/data/data/com.termux/files/home/Softwarw/perl-5.16.1/dist/Dumpvalue'
./miniperl -Ilib make_ext.pl dist/Env/pm_to_blib MAKE=make LIBPERL_A=libperl.a
	Making Env (all)
make[1]: Entering directory '/data/data/com.termux/files/home/Softwarw/perl-5.16.1/dist/Env'
Makefile out-of-date with respect to ../../lib/Config.pm ../../config.h
Cleaning current config before rebuilding Makefile...
make -f Makefile.old clean > /dev/null 2>&1
../../miniperl "-I../../lib" "-I../../lib" Makefile.PL "INSTALLDIRS=perl" "INSTALLMAN1DIR=none" "INSTALLMAN3DIR=none" "PERL_CORE=1" "LIBPERL_A=libperl.a"
Writing Makefile for Env
==> Your Makefile has been rebuilt. <==
==> Please rerun the make command.  <==
false
make[1]: *** [Makefile:728: Makefile] Error 1
make[1]: Leaving directory '/data/data/com.termux/files/home/Softwarw/perl-5.16.1/dist/Env'
 make config PERL_CORE=1 LIBPERL_A=libperl.a failed, continuing anyway...
Making all in dist/Env
 make all PERL_CORE=1 LIBPERL_A=libperl.a
make[1]: Entering directory '/data/data/com.termux/files/home/Softwarw/perl-5.16.1/dist/Env'
Skip ../../lib/Env.pm (unchanged)
make[1]: Leaving directory '/data/data/com.termux/files/home/Softwarw/perl-5.16.1/dist/Env'
./miniperl -Ilib make_ext.pl ext/Errno/pm_to_blib MAKE=make LIBPERL_A=libperl.a
	Making Errno (all)
make[1]: Entering directory '/data/data/com.termux/files/home/Softwarw/perl-5.16.1/ext/Errno'
Makefile out-of-date with respect to ../../lib/Config.pm ../../config.h
Cleaning current config before rebuilding Makefile...
make -f Makefile.old clean > /dev/null 2>&1
../../miniperl "-I../../lib" "-I../../lib" Makefile.PL "INSTALLDIRS=perl" "INSTALLMAN1DIR=none" "INSTALLMAN3DIR=none" "PERL_CORE=1" "LIBPERL_A=libperl.a"
Writing Makefile for Errno
==> Your Makefile has been rebuilt. <==
==> Please rerun the make command.  <==
false
make[1]: *** [Makefile:737: Makefile] Error 1
make[1]: Leaving directory '/data/data/com.termux/files/home/Softwarw/perl-5.16.1/ext/Errno'
 make config PERL_CORE=1 LIBPERL_A=libperl.a failed, continuing anyway...
Making all in ext/Errno
 make all PERL_CORE=1 LIBPERL_A=libperl.a
make[1]: Entering directory '/data/data/com.termux/files/home/Softwarw/perl-5.16.1/ext/Errno'
../../miniperl "-I../../lib" "-I../../lib" Errno_pm.PL Errno.pm
No error definitions found at Errno_pm.PL line 210.
make[1]: *** [Makefile:424: Errno.pm] Error 2
make[1]: Leaving directory '/data/data/com.termux/files/home/Softwarw/perl-5.16.1/ext/Errno'
Unsuccessful make(ext/Errno): code=512 at make_ext.pl line 466.
make: *** [makefile:663: ext/Errno/pm_to_blib] Error 2
```


Am I make something wrong or I can't make the perl with this way?

--- a/packages/perl/build.sh
+++ b/packages/perl/build.sh
@@ -39,36 +39,38 @@ termux_step_post_get_source() {
 termux_step_configure() {
 	export PATH=$PATH:$TERMUX_STANDALONE_TOOLCHAIN/bin
 
-	ORIG_AR=$AR; unset AR
-	ORIG_AS=$AS; unset AS
-	ORIG_CC=$CC; unset CC
-	ORIG_CXX=$CXX; unset CXX
-	ORIG_CPP=$CPP; unset CPP
-	ORIG_CFLAGS=$CFLAGS; unset CFLAGS
-	ORIG_CPPFLAGS=$CPPFLAGS; unset CPPFLAGS
-	ORIG_CXXFLAGS=$CXXFLAGS; unset CXXFLAGS
-	ORIG_LDFLAGS=$LDFLAGS; unset LDFLAGS
-	ORIG_RANLIB=$RANLIB; unset RANLIB
-	ORIG_LD=$LD; unset LD
-
 	# Since we specify $TERMUX_PREFIX/bin/sh below for the shell
 	# it will be run during the build, so temporarily (removed in
 	# termux_step_post_make_install below) setup symlink:
 	rm -f $TERMUX_PREFIX/bin/sh
 	ln -s /bin/sh $TERMUX_PREFIX/bin/sh
 
-	cd $TERMUX_PKG_BUILDDIR
-	$TERMUX_PKG_SRCDIR/configure \
-		--target=$TERMUX_HOST_PLATFORM \
-		-Dosname=android \
-		-Dsysroot=$TERMUX_STANDALONE_TOOLCHAIN/sysroot \
-		-Dprefix=$TERMUX_PREFIX \
-		-Dsh=$TERMUX_PREFIX/bin/sh \
-		-Dcc="$ORIG_CC" \
-		-Dld="$ORIG_CC -Wl,-rpath=$TERMUX_PREFIX/lib -Wl,--enable-new-dtags" \
-		-Dar="$ORIG_AR" \
-		-Duseshrplib \
-		-Dusethreads
+	(
+		ORIG_AR=$AR; unset AR
+		ORIG_AS=$AS; unset AS
+		ORIG_CC=$CC; unset CC
+		ORIG_CXX=$CXX; unset CXX
+		ORIG_CPP=$CPP; unset CPP
+		ORIG_CFLAGS=$CFLAGS; unset CFLAGS
+		ORIG_CPPFLAGS=$CPPFLAGS; unset CPPFLAGS
+		ORIG_CXXFLAGS=$CXXFLAGS; unset CXXFLAGS
+		ORIG_LDFLAGS=$LDFLAGS; unset LDFLAGS
+		ORIG_RANLIB=$RANLIB; unset RANLIB
+		ORIG_LD=$LD; unset LD
+
+		cd $TERMUX_PKG_BUILDDIR
+		$TERMUX_PKG_SRCDIR/configure \
+			--target=$TERMUX_HOST_PLATFORM \
+			-Dosname=android \
+			-Dsysroot=$TERMUX_STANDALONE_TOOLCHAIN/sysroot \
+			-Dprefix=$TERMUX_PREFIX \
+			-Dsh=$TERMUX_PREFIX/bin/sh \
+			-Dcc="$ORIG_CC" \
+			-Dld="$ORIG_CC -Wl,-rpath=$TERMUX_PREFIX/lib -Wl,--enable-new-dtags" \
+			-Dar="$ORIG_AR" \
+			-Duseshrplib \
+			-Dusethreads
+	)
 }
 
 termux_step_post_make_install() {

--- a/packages/perl/build.sh
+++ b/packages/perl/build.sh
@@ -11,7 +11,7 @@ TERMUX_PKG_MAINTAINER="@termux"
 # - subversion
 TERMUX_PKG_VERSION=(5.34.0
                     1.3.6)
-TERMUX_PKG_REVISION=2
+TERMUX_PKG_REVISION=3
 TERMUX_PKG_SHA256=(551efc818b968b05216024fb0b727ef2ad4c100f8cb6b43fab615fa78ae5be9a
                    4010f41870d64e3957b4b8ce70ebba10a7c4a3e86c5551acb4099c3fcbb37ce5)
 TERMUX_PKG_SRCURL=(http://www.cpan.org/src/5.0/perl-${TERMUX_PKG_VERSION}.tar.gz
@@ -67,7 +67,8 @@ termux_step_configure() {
 		-Dcc="$ORIG_CC" \
 		-Dld="$ORIG_CC -Wl,-rpath=$TERMUX_PREFIX/lib -Wl,--enable-new-dtags" \
 		-Dar="$ORIG_AR" \
-		-Duseshrplib
+		-Duseshrplib \
+		-Dusethreads
 }
 
 termux_step_post_make_install() {

